fastly.yaml

```diff
@@ -0,0 +1,4 @@
+---
+vhost:
```

Comment:
Tricky one. I couldn't think of the right language so I was deliberately vague. The issue with `service` is that's a name Fastly uses for a specific CDN configuration. What we want is something larger than a service - a `configuration` ?